From 8d2e12a0233e5ecdeb83c86a6c7e15ad6c907468 Mon Sep 17 00:00:00 2001
From: Saahil Tomar <quic_saahtoma@quicinc.com>
Date: Thu, 5 Oct 2023 14:24:00 +0530
Subject: [PATCH] arm64: dts: qcom: ipq5332: Add build support for RDP484

This change adds MI01.2-C2 (RDP484) which is a derivative of RDP441
with PON SFP connected and ethernet SFP plugged out

Change-Id: I14e95e3ea92ce10f117c0f8edf00d7ab135a34af
Signed-off-by: Saahil Tomar <quic_saahtoma@quicinc.com>
---
 arch/arm/boot/dts/Makefile        | 1 +
 arch/arm64/boot/dts/qcom/Makefile | 1 +
 2 files changed, 2 insertions(+)

diff --git a/arch/arm/boot/dts/Makefile b/arch/arm/boot/dts/Makefile
index 8e944eecc7ec..fd5c83c23c5d 100644
--- a/arch/arm/boot/dts/Makefile
+++ b/arch/arm/boot/dts/Makefile
@@ -1085,6 +1085,7 @@ dtb-$(CONFIG_ARCH_QCOM) += \
 	ipq5332-rdp478.dtb \
 	ipq5332-rdp479.dtb \
 	ipq5332-rdp481.dtb \
+	ipq5332-rdp484.dtb \
 	ipq5332-db-mi01.1.dtb \
 	ipq5332-db-mi02.1.dtb \
 	ipq5332-db-mi03.1.dtb \
diff --git a/arch/arm64/boot/dts/qcom/Makefile b/arch/arm64/boot/dts/qcom/Makefile
index 9b4f6b1cde90..e7b90f8a4195 100644
--- a/arch/arm64/boot/dts/qcom/Makefile
+++ b/arch/arm64/boot/dts/qcom/Makefile
@@ -15,6 +15,7 @@ dtb-$(CONFIG_ARCH_QCOM)	+= ipq5332-rdp477.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= ipq5332-rdp478.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= ipq5332-rdp479.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= ipq5332-rdp481.dtb
+dtb-$(CONFIG_ARCH_QCOM)	+= ipq5332-rdp484.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= ipq5332-db-mi01.1.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= ipq5332-db-mi02.1.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= ipq5332-db-mi03.1.dtb
-- 
2.34.1

